build.sbt

```diff
@@ -45,23 +45,48 @@ lazy val microservice = (project in file("."))
 lazy val appDependencies: Seq[ModuleID] = compile ++ test
 
 
+
+
+lazy val hmrcBootstrapPlay26Version = "1.3.0"
+lazy val hmrcSimpleReactivemongoVersion = "7.22.0-play-26"
+lazy val hmrcHttpMetricsVersion = "1.6.0-play-26"
+lazy val hmrcReactiveMongoTestVersion = "4.16.0-play-26"
+lazy val hmrcTestVersion = "3.9.0-play-26"
+lazy val scalaJVersion = "2.4.1"
+lazy val scalatestPlusPlayVersion = "3.1.2"
+lazy val mockitoVersion = "1.10.19"
+lazy val wireMockVersion = "2.21.0"
+// we need to override the akka version for now as newer versions are not compatible with reactivemongo
+lazy val akkaVersion = "2.5.23"
+lazy val akkaHttpVersion = "10.0.15"
+
 lazy val compile = Seq(
   ws,
-  "uk.gov.hmrc" %% "bootstrap-play-25" % "5.1.0"
+  "uk.gov.hmrc" %% "bootstrap-play-26" % hmrcBootstrapPlay26Version
 )
 
 lazy val test = Seq(
-  "uk.gov.hmrc" %% "hmrctest" % "3.9.0-play-25" % "test",
-  "org.scalaj" %% "scalaj-http" % "2.4.0" % "test",
   "org.pegdown" % "pegdown" % "1.6.0" % "test",
+  "org.webjars" %% "webjars-play" % "2.6.3",
+  "uk.gov.hmrc" %% "hmrctest" % hmrcTestVersion % "test",
+  "org.scalaj" %% "scalaj-http" % scalaJVersion % "test",
+  "org.scalatestplus.play" %% "scalatestplus-play" % scalatestPlusPlayVersion % "test",
+  "org.mockito" % "mockito-core" % mockitoVersion % "test",
   "com.typesafe.play" %% "play-test" % PlayVersion.current % "test",
-  "com.github.tomakehurst" % "wiremock" % "1.58" % "test",
-  "org.scalatestplus.play" %% "scalatestplus-play" % "2.0.1" % "test",
-  "org.mockito" % "mockito-core" % "1.10.19" % "test",
+  "com.github.tomakehurst" % "wiremock-jre8" % wireMockVersion % "test",
   "info.cukes" %% "cucumber-scala" % "1.2.5" % "test",
   "info.cukes" % "cucumber-junit" % "1.2.5" % "test"
 )
 
+
+lazy val overrides = Set(
```

Comment:
I think this hasn't been added to the settings

Reply:
have fixed this